### PR TITLE
Translate aggregation to portuguese

### PR DESCRIPTION
**Pull request by pri-hoh**
Brazilian portuguese.

## Author: Priscila Hohberg (@pri-hoh)

- 

## Language: Portuguese

- 

## Terms defined: Aggregation

- 
- 
- 

## Editor

Assign the PR based on the language to the following person:

| Language   | Person        | 
|:----------:|:-------------:|
| Afrikaans  | @jsteyn       |
| English    | @zkamvar      |
| French     | @fmichonneau  |
| Portuguese | @beatrizmilz  |
| Spanish    | @ian-flores   |

### CODE REVIEW

File: glossary.yml [hunk 1]
```diff
@@ -181,6 +181,11 @@
     def: >
       Om 'n groep waardes in een te kombineer bv. om 'n versameling getalle op
       te som of om 'n versameling alfabetiese stringe aaneen te skakel.
+  pt:
+    term: "agregação"
+    def: >
+      Combina vários valores em um, por exemplo, somando um conjunto numérico ou concatenando 
+      um conjunto de caracteres.
```
Comment (beatrizmilz) [inline glossary.yml:188]:
```suggestion
      Combinar vários valores em um, por exemplo, somando um conjunto numérico ou concatenando 
      um conjunto de caracteres.
```

**Comment by beatrizmilz** [comment]
Oi Pri! Acabei de ver que esse termo foi traduzido primeiro no #128 !

**Comment by beatrizmilz** [comment]
@fmichonneau I labeled it as 'Hacktoberfest-accepted' because is everything ok with this PR but someone else translated earlier the same term. Is that ok?

**Review by fmichonneau — APPROVED** [review]

**Comment by fmichonneau** [comment]
@beatrizmilz yes, that's fine. This is not needed though as the repository is labeled with "Hacktoberfest" so any pull requests made on the repository are elligible for the event. Thanks

**Comment by beatrizmilz** [comment]
Thank you both! @fmichonneau  @pri-hoh

**Pull request merged**